### PR TITLE
replace deprecated request pkg

### PR DESCRIPTION
**Pull request by jimmytsang**
replace the `request` package with `axios`

### CODE REVIEW

File: ui/package.json [hunk 1]
```diff
@@ -97,7 +96,7 @@
         "redux-devtools-extension": "2.13.9",
         "redux-mock-store": "1.5.4",
         "redux-thunk": "2.4.1",
-        "request": "2.88.2",
+        "axios": "1.4.0",
```
Comment (jimmytsang) [inline ui/package.json:99]: Considered using native nodejs functions but ran into connectivity errors that were not easily resolvable.
I believe this current solution using `axios` is appropriate for the time being as it involves very little code change and is a popular [npm package](https://www.npmjs.com/package/axios) that has a very active community supporting it.

File: ui/src/rdl-rest.js [hunk 2]
```diff
@@ -17,6 +17,7 @@
 
 var url = require('url');
 var path = require('path');
+var axios = require('axios');
 var request = require('request');
```
Comment (chandrasekhar1996) [inline ui/src/rdl-rest.js:21]: need to remove this

**Comment by chandrasekhar1996** [comment]
package-lock.json will also need to be updated

File: ui/src/rdl-rest.js (hunk 2 above)
Comment (jimmytsang) [reply ui/src/rdl-rest.js]: removed

**Review by chandrasekhar1996 — APPROVED** [review]

**Pull request merged**